Update.sh

```diff
@@ -45,30 +45,28 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 pushd "$SCRIPT_DIR" >/dev/null
 
 CONTENT_FOLDER="${SCRIPT_DIR}/Unreal/CarlaUE4/Content/Carla"
+BACKUP_CONTENT_FOLDER="${CONTENT_FOLDER}_$(date +%Y%m%d%H%M%S)"
+SCOOMATIC_ARCHIVE=Scoomatic/Carla.tar.gz
 
 CONTENT_ID=$(tac $SCRIPT_DIR/Util/ContentVersions.txt | egrep -m 1 . | rev | cut -d' ' -f1 | rev)
 CONTENT_LINK=http://carla-assets.s3.amazonaws.com/${CONTENT_ID}.tar.gz
 
 VERSION_FILE="${CONTENT_FOLDER}/.version"
 
 function download_content {
-  if [[ -d "$CONTENT_FOLDER" ]]; then
-    echo "Backing up existing Content..."
-    mv -v "$CONTENT_FOLDER" "${CONTENT_FOLDER}_$(date +%Y%m%d%H%M%S)"
+
+  echo "ensuring there's an empty $CONTENT_FOLDER dir"
+  if [ -d "$CONTENT_FOLDER" ]; then
+    mv "$CONTENT_FOLDER" "$BACKUP_CONTENT_FOLDER"
   fi
   mkdir -p "$CONTENT_FOLDER"
-  mkdir -p Content
-  if hash aria2c 2>/dev/null; then
-    echo -e "${CONTENT_LINK}\n\tout=Content.tar.gz" > .aria2c.input
-    aria2c -j16 -x16 --input-file=.aria2c.input
-    rm -f .aria2c.input
-  else
-    wget -c ${CONTENT_LINK} -O Content.tar.gz
-  fi
-  tar -xvzf Content.tar.gz -C Content
-  rm Content.tar.gz
-  mv Content/* "$CONTENT_FOLDER"
-  rm -rf Content
+
+  echo "downloading contents from $CONTENT_LINK and unpacking them to $CONTENT_FOLDER"
+  wget -qO- "$CONTENT_LINK" | tar xz -C "$CONTENT_FOLDER"
+
+  echo "unpacking Scoomatic contents (with overwrite)"
+  tar -xzf "$SCOOMATIC_ARCHIVE" -C "$CONTENT_FOLDER"
+
```

Comment:
Just for clarification. This aspect of the `Update.sh` ensures that you store the scoomatic contents temporarily in the `SCOOMATIC_ARCHIVE` and that the Content can be restored after Updating the assets?

Reply:
Yes, that's the whole trick.

Reply:
The only file that's really overwritten seems to be the vehicle factory uasset.

If Update.sh registers another vehicle from the official resources, it would not appear because the vehicle factory from the Scoomatic archive doesn't contain it. 

So there needs to be some manual merging process to fix this. But the build would not break (I've tested this); the new vehicle is just ignored. Another problem might be when an official vehicle gets removed (but that's very unlikely).

Reply:
In my experience, no vehicle has ever been removed. I am happy with this implementation, and I am aware of its limitations. Thanks.